apparun/impact_methods.py

```diff
@@ -9,8 +9,6 @@ class MethodFullName(str, Enum):
     So far, only some ReCiPe methods, and all EF v3.0 impact methods are supported.
     """
 
-    _settings_ = NoAlias
```

Comment:
why removing?

Reply:
to enable by-value lookups
`NoAlias` is not required as there are no value duplicates in `MethodFullName`